src/components/home/DrawerContent.js

```diff
@@ -0,0 +1,140 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import PropTypes from 'prop-types';
+
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import MenuList from '@material-ui/core/MenuList';
+import Divider from '@material-ui/core/Divider';
+import { withStyles } from '@material-ui/core';
+import ExpandLess from '@material-ui/icons/ExpandLess';
+import ExpandMore from '@material-ui/icons/ExpandMore';
+import Collapse from '@material-ui/core/Collapse';
+import Typography from '@material-ui/core/Typography';
+import Hidden from '@material-ui/core/Hidden';
+
+import {
+  searchEnvironmentsByTopic,
+  resetCategorizedEnvironments,
+} from '../../actions/environments';
+import DrawerHead from './DrawerHead';
+import ChildListElement from './ChildListElement';
+import SearchBar from '../header/SearchBar';
+
+const drawerWidth = 240;
+const styles = theme => ({
+  root: {
+    width: drawerWidth,
+    flexShrink: 0,
+    paddingTop: '0',
+    paddingBottom: '0',
+  },
+  title: {
+    margin: theme.spacing.unit * 2,
+    marginTop: theme.spacing.unit * 6,
+  },
+});
+
+class DrawerContent extends Component {
+  constructor(props) {
+    super(props);
+    const parentTopics = this.props.topics.filter(topic => !topic.parentTopic);
+    this.state = {
+      open: Array.apply(null, Array(parentTopics.length)).map(() => false),
```

Comment:
`open` is vague, try to give this a more descriptive name.

Reply:
done